invokeai/app/invocations/image.py

```diff
@@ -80,18 +80,19 @@ class LoadImageInvocation(BaseInvocation):
 
     # fmt: off
     type: Literal["load_image"] = "load_image"
-
     # Inputs
-    image_type: ImageType = Field(description="The type of the image")
-    image_name:       str = Field(description="The name of the image")
+    image: ImageField = Field(description="The type of the image")
```

Comment:
This may make it difficult to use in the CLI. Note that this node is effectively just "create an image field".

Reply:
It's useful in the UI when you want to pass the same image to multiple image inputs

Reply:
You can't construct this data structure in the CLI though.

Reply:
Hmm. So there's no way to do img2img on the cli with a user-provided image?

Reply:
Maybe `Image loading` node fits here?
This node will load image and add to cache with this name, and it's output is `ImageField`.

Reply:
> It's useful in the UI when you want to pass the same image to multiple image inputs

If I understand correctly - if we have `ImageField` we already can pass it to multiple nodes, isn't it?
![image](https://user-images.githubusercontent.com/7768370/234124577-f9f81ae1-10bb-4c70-8258-b905ee70049f.png)

And to load new image to graph we using this node with simple values without `ImageField`.

Reply:
Yah load image is just to take a type and name and create an `ImageField`, which can then be passed to other things. Basically how you'd load an uploaded file into the pipeline.

Reply:
Even if LoadImage has a type and name as inputs instead of an ImageField, as you've suggested, it still doesn't let users on the cli take an image from anywhere on disk and use it. That use case is not addressed by LoadImage regardless.

That aside, it's much simpler for LoadImage to have an ImageField than a name and type.

Then the UI can present it like any other image field input (which allows you to drag an image from the gallery into it, or in the future, drag an image from the file system onto it to upload).

If it's just a type and name, the ui needs to have special handling for this specific node.

Reply:
*goes to look at load_image node*...

```py
# TODO: this isn't really necessary anymore
```

Well then... still concerned about how to do this with the UI, but maybe this node should just be deleted (and create a separate "param" node for images, like the int/float/string params, since that's what you seem to intend to use this like).

Reply:
Yeah that's how it's being used. @blessedcoolant lets follow that suggestion and make this a "param" node. 

We will have to figure out a CLI solution for loading an image from disk. May need to just have some nodes that are cli-only or ui-only

Reply:
That's what commands are for in the CLI. Already have a bunch of different commands supported =)